src/pages/NewTask.tsx

```diff
@@ -60,7 +60,7 @@ class NewTask extends React.Component<WrappedComponentProps, IState> {
         const { select_task } = taskInput[0] as { select_task: string };
 
         let promise = this.context.apiClient.startProcess(select_task, taskInput.slice(1)).then((process) => {
-            this.context.redirect(`/tasks?highlight=${process.id}`);
+            this.context.redirect(`/processes/${process.id}`);
```

Comment:
Not really in scope of this change, but perhaps we should also have a `/tasks/<pid>` route in the app and make tasks more distinct from processes (at least in the frontend, on the backend they're almost identica of course)

Reply:
yea, maybe, lets discuss next meeting.